docs/pnpm-lock.yaml

```diff
@@ -150,267 +150,362 @@ devDependencies:
     version: 3.12.6
 
 packages:
-
   /@babel/helper-string-parser@7.22.5:
-    resolution: {integrity: sha512-mM4COjgZox8U+JcXQwPijIZLElkgEpO5rsERVDJTc2qfCDfERyob6k5WegS14SX18IIjv+XD+GrqNumY5JRCDw==}
-    engines: {node: '>=6.9.0'}
+    resolution:
```

Comment:
Does it really make sense to format the lock-file?

Reply:
Not really, you're right. Will create a separate PR where I exclude this stuff